src/frontend-samples/validation-sample/ValidationClient.ts

```diff
@@ -0,0 +1,177 @@
+        return undefined;
+      });
+  }
+  // END API_EXAMPLE
+
+  // Retrieves a list of Design Validation runs for the project specified by the project id.
+  public static async getProjectValidationRuns(requestContext: AuthorizedClientRequestContext, projectId: string): Promise<any | undefined> {
+    const accessToken = await ValidationClient.getAccessToken();
+    if (accessToken === undefined)
+      return undefined;
+
+    const url = `https://api.bentley.com/projects/${projectId}/validation/runs`;
+    const options: RequestOptions = {
+      method: "GET",
+      headers: {
+        Prefer: "return=representation",
+        Authorization: accessToken.toTokenString(IncludePrefix.Yes),
+      },
+    };
+    return request(requestContext, url, options)
+      .then((resp: Response): string | undefined => {
+        if (resp.body === undefined) return undefined;
+        return resp.body;
+      }).catch((_reason: any) => {
+        return undefined;
+      });
+  }
+
+  // Gets the response body for the specified validation URL.
+  public static async getValidationUrlResponse(requestContext: AuthorizedClientRequestContext, url: string) {
+    if (url === undefined)
+      return undefined;
+
+    const accessToken = await ValidationClient.getAccessToken();
+    if (accessToken === undefined)
+      return undefined;
+
+    const options = {
+      method: "GET",
+      headers: {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        Authorization: accessToken.toTokenString(IncludePrefix.Yes),
+      },
+    };
+    return request(requestContext, url, options)
+      .then((resp: Response): any | undefined => {
+        return resp.body;
+      }).catch((_reason: any) => {
+        return undefined;
+      });
+  }
+
+  // Run a validation test for the specified test id and return the run id and URL link to monitor progress.
+  //
+  // NOTE: This API will not succeed within the frontend sample showcase since the user token does not have
+  //       modify access to the project.
+  //
+  //   Sample response:
+  //   {
+  //     "validationRunLink": {
+  //         "runId": "ZuO3OCC8sUuVcgeXz1Ih_cd5x2CBBWdIpYrvJ8Y-qIY",
+  //         "_links": {
+  //             "run": {
+  //                 "href": "https://api.bentley.com/validation/runs/ZuO3OCC8sUuVcgeXz1Ih_cd5x2CBBWdIpYrvJ8Y-qIY"
+  //             }
+  //         }
+  //     }
+  // }
+  // You can pass the URL returned in the href to the following function:
+  //   const response = ClashDetectionApis.getValidationUrlResponse(body._links.run.href);
+  public static async runValidationTest(requestContext: AuthorizedClientRequestContext, iModelId: string, testId: string) {
+    if (testId === undefined)
+      return undefined;
+    const accessToken = await ValidationClient.getAccessToken();
+    if (accessToken === undefined)
+      return undefined;
+
+    const url = `https://dev-api.bentley.com/validation/runs/test/${testId}`;
```

Comment:
```suggestion
    const url = `https://api.bentley.com/validation/runs/test/${testId}`;
```
Please remove references to dev